app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+
+  def new
+  end
+
+  def create
+    user = User.new(user_params)
+    if user.save!
+      session[:user_id] = user.id
+      redirect_to return_point, notice: "You are now signed in"
+    else
+      redirect_to new_user_path
+    end
+  end
+
+  def show
+    @user = current_user
+    @questions = @user.questions
```

Comment:
You don't need to use separate instance variables for data that is accessible via associations.